### ISSUE TITLE
To release a Windows version of the app

### ISSUE THREAD

**Issue by sallymoc** [qubic/wallet-app#44]

**Comment by sallymoc** [comment]
@Qubic-Hub  I understand you are normally running the app in Windows, however we are not releasing an app for this platform yet.
What is missing/blocking this?

**Comment by Qubic-Hub** [comment]
@sallymoc  I don't think that anything is blocking us from doing this. Last time I had asked , JT told that we don't need windows app that's why it's left out. 
Apart from that it's ready to compile and run

**Comment by Qubic-Hub** [comment]
Will include new .exe builds for later release

**Comment by sallymoc** [comment]
To ensure that your .exe file is trusted by Windows and users, we need to obtain and use a code signing certificate from a reputable Certificate Authority.
